puzzle.js

```diff
@@ -813,25 +824,25 @@ function PuzzleEntry(p) {
                 if (unselectableGivens) { td.classList.add("unselectable"); }
             }
             else {
-                cell.value = this.translate(shapeCh, shapeReplacements);
+                text.innerText = this.translate(shapeCh, shapeReplacements);
                 td.classList.add("given");
                 if (unselectableGivens) { td.classList.add("unselectable"); }
             }
 
             if (!td.classList.contains("unselectable")) {
-                cell.addEventListener("keydown",  e => { this.keyDown(e); });
-
+                td.tabIndex = -1;
```

Comment:
This makes it so we can't tab into the interactive elements, or tab from one interactive element to the next.

Can we make the very first td of every puzzle have tabIndex = 0? The rest should be -1

Reply:
I'll do this but TBH it doesn't seem to make any difference